Resources/clientCommandPerms.yml

```diff
@@ -46,22 +46,17 @@
     - showambient
     - showemergencyshuttle
     - zoom
-
-- Flags: MAPPING
-  Commands:
-    - showmarkers
     - showsubfloor
     - showsubfloorforever
-    - mapping
-    - toggleautosave
-    - toggledecals
+    - showmarkers
     - nodevis
     - nodevisfilter
 
-- Flags: ADMIN
+- Flags: MAPPING
   Commands:
-    - togglehealthoverlay
-    - togglelight
```

Comment:
is the toggle light removal from mapping intentional?

Reply:
this is already handled in engine command perms, `togglelight` is an engine command

Reply:
https://github.com/mirrorcult/space-station-14/blob/03cb8ca281af3cb3607d49d8a635285f578e8600/Resources/engineCommandPerms.yml#L29